tasks/main.yml

```diff
@@ -147,6 +147,9 @@
          (item.enabled|d(True) | bool) and
          (item.when|d(True) | bool))
 
+- name: Ensure that /etc/pki directory exists
+  file: path=/etc/pki state=directory
```

Comment:
You should use the YAML syntax (similar to other tasks in this file) and ensure that the corret user, group and permissions are set as well.